RELEASE_PROCESS.md

```diff
@@ -0,0 +1,44 @@
+# TektonCD CLI release process
+
+- You need to be a tektoncd-cli [OWNERS](OWNERS) to be able to push the new branch
+
+- You need to have your own tekton cluster, under `minikube`, `gcp` or other means.
+
+- Start reading this [README](tekton/README.md) and you can start a new release
+  with the [release.sh](tekton/release.sh) script.
+
+- Update the main [README.md](README.md) and change the version number in there (i.e: `v0.5.0`)
+  to the version you are releasing. Make a PR and get it merged ASAP.
+
+- When release is done in https://github.com/tektoncd/cli/releases edit the
+  release and change it from pre-release as released.
+
+- When it's released you can build the rpm according to this
+  [README](tekton/rpmbuild/README.md). Make sure you have done the PREREQ to be
+  able to upload it to the copr repository. This could take some time if copr
+  builder is busy.
+
+- you need to edit the Changelog according to the other templates in there,
+  usually you may want to do this in hackmd so you can have other cli
+  maintainers do this with you. an example
+  [here](https://hackmd.io/Gj9bxTh0Qn6kOaEx8gjsJg?edit).
```

Comment:
Try updating the url to https://hackmd.io/@chmouel/SykvrTGoB/

Reply:
lol i made a edit and pushed --force but forgot to save in my editor!